src/js/plugin/PluginEventProxy.js

```diff
@@ -11,7 +13,7 @@ var proxies = [];
 function registerProxy(proxy) {
   proxies.push(function () {
     proxy.store.on(proxy.storeEvent, () => {
-      PluginDispatcher.dispatch({
+      PluginDispatcher.dispatchNext({
```

Comment:
This prevents a race condition between the plugin and the DuplateRowsMixin that call FormActions.
Otherwise I get a "can't dispatch in a middle of a dispatch".